### ISSUE TITLE
Adding new users breaks

### ISSUE THREAD

**Issue by olivierjeunen** [david-cortes/ctpfrec#1]
When trying to add new users to a previously trained model, NumPy complains with:

```
   1813    if self.keep_data and (counts_df is not None):
-> 1814        for u in range(new_max_id):
   1815            items_this_user = counts_df.ItemId.values[counts_df.UserId == u]

TypeError: 'numpy.float64' object cannot be interpreted as an integer
```

It seems that new_max_id is casted to a numpy float somewhere along the way - even when the counts and words dataframes only contain integer identifiers.

Minimal code block to reproduce the error:

```python
import numpy as np
import pandas as pd
from ctpfrec import CTPF

# Dummy data
counts_df = pd.DataFrame([[0,0,1],[0,1,1]], columns = ['UserId','ItemId','Count'])
words_df = pd.DataFrame([[0,0,1],[0,1,1]], columns = ['ItemId','WordId','Count'])

# Fit model
recommender = CTPF(k = 5,
                   reindex = True)
recommender.fit(counts_df = counts_df,
                words_df = words_df)

# Generate new dummy user
counts_df_new = pd.DataFrame([[1,0,1],[1,1,1]], columns = ['UserId','ItemId','Count'])

# Add new dummy user !< This breaks
recommender.add_users(counts_df = counts_df_new)
```

I suspected this might have had something to do with the re-indexing, but when disabling this option in the instantiation of the CTPF object, the fit call complains with:
```
--> 896    items_intersect = np.in1d(items_words_df, items_counts_df)
    897    words_include = self._words_df.WordId.loc[np.in1d(self._words_df.ItemId, items_words_df[items_intersect])].unique()

NameError: name 'items_counts_df' is not defined
```

My NumPy version is 1.18.1.

**Comment by david-cortes** [comment]
Thanks for the bug report, could reproduce both issues, will fix.

**Comment by david-cortes** [comment]
Fixed now, please update to version 0.1.4 and reopen if the issue persists.
